manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager-operator
+  labels:
+    k8s-app: cloud-manager-operator
+spec:
+  selector:
+    matchLabels:
+      k8s-app: cloud-manager-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: cloud-manager-operator
+    spec:
+      priorityClassName: system-node-critical
```

Comment:
What are the options here? Does this mean this component is critical if we want nodes to work correctly?

Reply:
According to [doc](https://github.com/openshift/openshift-docs/blob/3f1927c4088f97385bd44b53ee0b62480958a67f/modules/nodes-pods-priority-about.adoc) there are a couple. I'm following the [practices](https://github.com/openshift/machine-api-operator/blob/331808946db5135185c82792b10ee430ec4b46c1/install/0000_30_machine-api-operator_11_deployment.yaml#L21) of both `MAO` and `KCM` here.